bindings/nodejs/generated.d.ts

```diff
@@ -32,6 +32,35 @@ export interface ListOptions {
   limit?: number
   recursive?: boolean
 }
+export interface OpWriteOptions {
```

Comment:
Those change should not be include in this PR.

Reply:
Hi, I will rebase this PR, and force push once the depend-on PR has merged, that should be a problem

Reply:
Thank you!